libs/langchain/langchain/chains/api/base.py

```diff
@@ -40,6 +73,19 @@ class APIChain(Chain):
     api_docs: str
     question_key: str = "question"  #: :meta private:
     output_key: str = "output"  #: :meta private:
+    limit_to_domains: Optional[Sequence[str]] = ()
```

Comment:
doe we need a default value? we raise a value if one's not explicitly specified anyways

Reply:
updated -- but no functional change either way